dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/HTMLPageAssetRenderedAPI.java

```diff
@@ -93,5 +67,10 @@ public String getPageHtml(final HttpServletRequest request, final HttpServletRes
      * @param pageUri
      * @return
      */
-     PageMode getDefaultEditPageMode(final User user, final HttpServletRequest request, final String pageUri);
+     PageMode getDefaultEditPageMode(
+             final User user,
+             final HttpServletRequest request,
+             final String pageUri,
+             final HttpServletResponse response
```

Comment:
Do we need the response here?

Reply:
why do we need to remove the response, last thing I have learned is:
if you sent the request, it is a good idea to send also the response, may be  the requirements now does not need the response, but it could be in the future and the refactor is painful